test/unit/specs/children.spec.js

```diff
@@ -0,0 +1,78 @@
+import Vue from 'vue'
+import VueRouter from '../../../src/index'
+
+Vue.use(VueRouter)
+
+describe('currentRoute', () => {
+  describe('children[]', () => {
```

Comment:
I don't think a describe is necessary here. a test with _should have children_ and another with _should have parent.children_ is enough